irispy/spectrograph.py

```diff
@@ -15,9 +15,9 @@
 from astropy import constants
 from scipy import interpolate
 from sunpy.time import parse_time
-
+from sunpy.net.vso import attrs as a
```

Comment:
I think we should have more descriptive variable names than ```a```.  But ```attrs``` is also too generic.  Perhaps it would be better to do ```from sunpy.net import vso.attrs```.